server/db/migrate/20150216190947_add_rhev_params.rb

```diff
@@ -0,0 +1,16 @@
+class AddRhevParams < ActiveRecord::Migration
```

Comment:
I ran the following in postgresql:
```
drop fusor_deployments;
delete from schema_migrations where version = '20150216190947';
delete from schema_migrations where version = '20150123165155';
commit;
```
Then I reran ```rake db:migrate``` from the foreman directory. Changes will be pushed soon.